reana_client/api/client.py

```diff
@@ -250,12 +250,12 @@ def upload_file(workflow_id, file_, file_name, access_token):
     :param file_name: name of a file that will be uploaded.
     :param access_token: access token of the current user.
     """
+    from reana_client.cli.utils import get_api_url
```

Comment:
Wouldn't it be better in the `reana_client/utils.py`? Since both `reana_client/api/*.py` and `reana_client/cli/*.py` will need it:
```
~/reana_client/
...
├── utils.py
...
├── api
│   └── client.py
...
├── cli
│   └── workflows.py
```